.scripts/setup.sh

```diff
@@ -1,21 +1,9 @@
 #!/usr/bin/env bash
 
-if [ "$CI" = true ]; then
-	printf "\n\n Install Slather - Code coverage tool \n"
-	gem install slather
-
-	printf "\n\n Install Xcpretty - Prettify xcodebuild logs \n"
-	gem install xcpretty -N --no-document
-
-	printf "\n\n Install Cocoapods - Dependencies manager \n"
-	gem install cocoapods -v '1.8.4'
-fi
-
-printf "\n\n Setup env variables \n"
-
-BUILD_DEST_IOS="platform=iOS Simulator,OS=13.4,name=iPhone 11 Pro"
-BUILD_DEST_PAIRED="platform=iOS Simulator,OS=13.4,name=iPhone 11 Pro"
-BUILD_DEST_WATCH="platform=watchOS Simulator,OS=6.2,name=Apple Watch Series 5 - 44mm"
+# Set testing environment variables
```

Comment:
Can we not set these env vars in the Github Action workflow?

Reply:
We could but these couple of scripts are very useful when we want to run tests and builds from command line too.
Calling `. .scripts/setup.sh` it sets all the variables, and then we can use the various `.scripts/test_ios_demo.sh` to check that everything is ok.